### ISSUE TITLE
Not generating phone numbers

### ISSUE THREAD

**Issue by franticstone** [N3MTV/gcphone#308]
This seems to be a topic thats raised (from my searching. I couldnt get this working at all then I noticed this line 

https://github.com/N3MTV/gcphone/blob/7837f4f034519dc63b02ff012a3df3adc07321c0/resources/gcphone/server/server.lua#L566

The event you wrote is ```es:playerLoaded``` when in fact that never fires so never generates a number. I changed it to ```esx:playerLoaded``` and it generated the number first time.

**Comment by franticstone** [comment]
I have made a pull request for you

https://github.com/N3MTV/gcphone/pull/309

**Comment by rex2630** [comment]
Your solution isnt good at all. Mine is a lot better: https://github.com/N3MTV/gcphone/pull/305

**Comment by franticstone** [comment]
im fixing one issue not everything else

**Comment by rex2630** [comment]
Yeah, but your PR is not needed anymore, since i fixed lot of more issues

**Comment by franticstone** [comment]
yours has many mines has one. if they want it they want it if they dont they dont. no need making it a competition
